### DISCUSSION TITLE
Account Issue

### DISCUSSION THREAD

**Discussion by verstilevoyager** [vercel/community#3836] [Account Issue] [answered]
### Question

I need guidance as I am getting this error "This user account is blocked" when I do a versal signup with github. 
While my github account is also complete.

**Comment by amyegan** [comment, marked as answer]
Sorry you're having trouble signing up for a new account. We can't safely discuss account information in public because of security concerns. You may have received an email with more details about the error and instructions on next steps. 

For more info, please check this guide: [Why am I unable to login or signup to the Vercel platform?](https://vercel.com/guides/why-can-i-not-signup)

**Comment by verstilevoyager** [comment]
I need guidance as I am getting this error "This user account is blocked"
when I do a versal signup with github.
While my github account is also complete.
